lib/influxql/quote.ex

```diff
@@ -66,7 +59,12 @@ defmodule InfluxQL.Quote do
     end
   end
 
-  def identifier(identifier), do: identifier |> Kernel.to_string() |> identifier()
+  def identifier(identifier) when is_atom(identifier),
+    do: identifier |> Kernel.to_string() |> identifier()
```

Comment:
Just having this one guard will miss some possible values (not directly covered in the tests, but acceptable) like `charlist` or `number`. A pretty full list should be extractable from `value/1`.